src/Common/Priority.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include <base/types.h>
+
+/// Common type for priority values.
+/// Separate type (rather than `Int64` is used just to avoid implicit conversion errors and to default-initialize
+struct Priority {
```

Comment:
Probably it would be better to make this class `enum` ? And have constants like `Priority::Normal`, `Priority::High` and `Priority::Turbo` ?

Reply:
I would like to, but we already have pattern with a cycle with increasing priorities in one place. But maybe we can rework it...